src/tests/setup.ts

```diff
@@ -0,0 +1,38 @@
+import { NodeGetParams, NodeSearchParams } from 'nodes/types';
+import SiteSails from '../index';
+
+const ss = new SiteSails({
+  apiUrl: 'https://api.sitesails.com/api/v1',
+  // TODOD admin COPIED FROM CGH
+  adminApiKey: '0fe40734-ae13-4eee-9782-df342c203bda',
+  publicApiKey: '66461de9-b081-43ed-87c4-7fa098829251',
+});
+
+export const fetchNodes = async (params: NodeSearchParams = {}) => {
```

Comment:
we probably don't need these helper wrappers but can simply export `ss` from here and use it directly in tests - these wrappers aren't adding anything other than re-calling the same methods